components/expand-collapse/expand-collapse-content.js

```diff
@@ -44,18 +46,34 @@ class ExpandCollapseContent extends LitElement {
 
 			.d2l-expand-collapse-content-container {
 				display: none;
+				opacity: 0;
 				overflow: hidden;
-				transition: height 400ms cubic-bezier(0, 0.7, 0.5, 1);
+				transition:
+					height var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function),
+					opacity var(--d2l-expand-collapse-content-transition-duration) var(--d2l-expand-collapse-content-transition-function);
 			}
 
 			.d2l-expand-collapse-content-container:not([data-state="collapsed"]) {
 				display: block;
 			}
 
+			.d2l-expand-collapse-content-container[data-state="collapsed"] {
+				opacity: 0;
+			}
+
+			.d2l-expand-collapse-content-container[data-state="collapsing"] {
+				opacity: 0;
+			}
```

Comment:
I feel like some of these selectors could be reworked to eliminate the `:not()`, but moving things around obviously carries some risk. I think we can at least collapse some of this duplication though.
```suggestion
			.d2l-expand-collapse-content-container[data-state="collapsed"],
			.d2l-expand-collapse-content-container[data-state="collapsing"] {
				opacity: 0;
			}
```